includes/abstracts/abstract-wp-job-manager-email-template.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Abstract class for an email notification built using templates.
+ *
+ * @package wp-job-manager
+ *
+ * @since 1.31.0
+ */
+
+abstract class WP_Job_Manager_Email_Template extends WP_Job_Manager_Email {
```

Comment:
Just FYI, `WP_Job_Manager_Email` class is perfectly fine, but wanted to point out that my addon uses `WP_Job_Manager_Emails` ... just something to keep in mind

Reply:
Noted 👍